docs/sphinx/development/continuous_integration.rst

```diff
@@ -61,6 +61,8 @@ Android cross-compilation
 Image                                      APIs        ABIs                                Compilers
 ========================================== =========== =================================== =============
 rocstreaming/toolchain-linux-android:ndk21 21-29       armeabi-v7a, arm64-v8a, x86, x86_64 clang-9.0.8
+rocstreaming/env-android:jdk8              21-29       armeabi-v7a, arm64-v8a, x86, x86_64 clang-9.0.8
+rocstreaming/env-android:jdk11             21-29       armeabi-v7a, arm64-v8a, x86, x86_64 clang-9.0.8
```

Comment:
I suggest to extract these two into a separate table "Full Android environment", to distinguish them from toolchain-linux-android. And we can rename "Linux cross-compilation" and "Android cross-compilation" tables to "Linux toolchains" and "Android toolchains" to make the difference more clear.

Also, it seems that listing APIs, ABIs, and compilers for env-android doesn't make sense because it may install any needed version? If so, I'd suggest to remove those columns. Instead, we can add a column with JDK version.

Reply:
Ok :+1: